indent/javascript.vim

```diff
@@ -176,6 +180,12 @@ function GetJavascriptIndent()
 
   let b:js_cache[:2] = [v:lnum,num,line('.') == v:lnum ? b:js_cache[2] : col('.')]
 
+  if s:xml && getline(b:js_cache[1])[b:js_cache[2]-1] == '(' && search('^\s*\S\%' . v:lnum .'l') &&
```

Comment:
While running XmlIndentGet returns reasonable indentation for the line, I wasn't able to write a jsx code that satisfies conditions in this if. So it applies normal javascript indentation.